### PR TITLE
[IMP]10.0 Update README.rst

### PR DESCRIPTION
**Pull request by ovnicraft**
Add simple note about how to configure products for this module

### CODE REVIEW

**Comment by ovnicraft** [comment]
Any review here ? @pedrobaeza

File: mrp_mto_with_stock/README.rst [hunk 1]
```diff
@@ -34,6 +34,8 @@ If you want to use the second mode, based on forecast quantity
 #. In the view form go to the tab *Warehouse Configuration* and set the *MRP 
    MTO with forecast stock*. You still need to configure the products
    like described in last step.
+   
+* NOTE: product you want follow this behaviour must not have configured routes.
```
Comment (pedrobaeza) [inline mrp_mto_with_stock/README.rst:38]: s/want follow/want to follow

**Comment by pedrobaeza** [comment]
I can't review about the content itself, as I don't know this module in depth.

File: mrp_mto_with_stock/README.rst (hunk 1 above)
Comment (LoisRForgeFlow) [reply mrp_mto_with_stock/README.rst:38]: That sentence is not true... actually you need some route configured to fulfill the needs that cannot be covered by the stock onhand.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by ovnicraft** [comment]
My fault, is MUST have as @lreficent mentions, i open a new PR for this by any reason github cant find my fork.

**Pull request closed without merging**